include/af/memory.h

```diff
@@ -174,6 +174,12 @@ AFAPI af_err af_release_memory_manager(af_memory_manager handle);
 /// \returns AF_SUCCESS
 AFAPI af_err af_set_memory_manager(af_memory_manager mgr);
 
+AFAPI af_err af_memory_manager_get_payload(af_memory_manager mgr,
+                                           void** payload);
+
+AFAPI af_err af_memory_manager_set_payload(af_memory_manager mgr,
+                                           void* payload);
```

Comment:
Ah, [glfwSetWindowUserPointer](https://www.glfw.org/docs/latest/group__window.html#ga3d2fc6026e690ab31a13f78bc9fd3651) style. I like this one.